examples/grafana-agent-auto-instrumentation/ebpf-otel/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenTelemetry eBPF Profiler Example
```

Comment:
We use sentence case instead of title case in headings. For more information: https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/
```suggestion
# OpenTelemetry eBPF profiler example
```